tests/models/test_model.py

```diff
@@ -325,3 +367,27 @@ def test_model_uuid():
     m_dict.pop("model_uuid")
     m4 = Model.from_dict(m_dict)
     assert m4.model_uuid is None
+
+
+def test_validate_schema(sklearn_knn_model, iris_data, tmpdir, model_path):
+    sk_model_path = os.path.join(str(tmpdir), "knn.pkl")
+    with open(sk_model_path, "wb") as f:
+        pickle.dump(sklearn_knn_model, f)
+
+    model_config = Model(run_id="test", artifact_path="testtest")
+    X, y = iris_data
+    signature = infer_signature(X, y)
+    mlflow.pyfunc.save_model(
+        path=model_path,
+        data_path=sk_model_path,
+        loader_module=__name__,
+        code_path=[__file__],
+        mlflow_model=model_config,
+        signature=signature,
+    )
```

Comment:
```suggestion
    sk_model_path = os.path.join(str(tmpdir), "knn.pkl")
    X, y = iris_data
    signature = infer_signature(X, y)
    with mlflow.start_run():
    mlflow.sklearn.save_model(
        sklearn_knn_model,
        sk_model_path,
        signature=signature,
    )
```

Can we use `lflow.sklearn.save_model`?

Reply:
Sure.